readme.md

```diff
@@ -1,175 +1,7 @@
-
-### ads
-
-Im Ordner ads befinden sich folgende vordefinierte Vorlagen, welche nicht angepasst werden müssen (Anpassungen erfolgen automatisch):
-
-* Deckblatt
-* Eigenständigkeitserklärung
-* Sperrvermerk
-* LaTeX Document Header
-
-### lang
-
-Im Ordner lang befinden sich alle nötigen Übersetzungen.
-
-### settings
-
-Im Ordner settings gibt es zwei Datein:
-
-* general.tex
-* document.tex
-
-In der Datei general.tex sind Grundlegende Einstellungen vordefiniert, welche nicht geändert werden müssen.
-
-In der Datei document.tex müssen einige Angaben über die zu schreibende Arbeit gemacht werden:
-
-| Variable | Beschreibung | Mögliche Werte |
-| -------- | ------------ | -------------- |
-| documentLanguage| Sprache der Arbeit | de<br/> en |
-| documentType | Art der Arbeit | T2\\_1000 Projektarbeit (Semester 1 & 2) <br/> T2\\_2000 Projektarbeit (Semester 3 & 4) <br/> T2\\_3100 Studienarbeit (Semester 5) <br/> T2\\_3300 Bachelorarbeit |
-| multipleAuthors | Wurde die Arbeit von mehreren Autoren verfasst? | true<br/> false |
-| documentAuthor | Autor der Arbeit | |
-| documentTitle | Titel der Arbeit | |
-| documentPeriod | Dauer der Arbeit | |
-| matriculationNumber | Matrikelnummer des Autors | |
-| locationUniversity | Standort der DHBW | Heidenheim |
-| department | Fakultät der DHBW in der sich der Autor befindet | |
-| course | Kurs in dem sich der Autor befindet | |
-| degree | Abschluss, welcher mit dieser Arbeit angestrebt wird | Bachelor of Science (INF2014-MI - INF2016-MI) <br/> Bachelor of Engineering (INF2014-IA/IM - INF2016-IA/IM) <br/> Bachelor of Science  (INF2017-IM/MI/IA) |
-|releaseDate | Abgabedatum | |
-| releaseLocation | Abgabeort | Heidenheim |
-| companyName | Name des Unternehmens in dem der Autor angestellt ist | |
-| companyLocation | Firmensitz | |
-| tutor | Betrieblicher Betreuer der Arbeit | |
-| evaluator | Zweitkorrektor der Arbeit | |
-
-### content
-
-### images
-
-
-# Document Types
-
-Das Templatet bietet die forlgenden verschiedenen Document Types an:
-
-* T2_1000 Project Thesis (Semester 1 & 2)
-* T2_2000 Project Thesis (Semester 3 & 4)
-* T2_3100 Seminar Paper (Semester 5 & 6)
-* T2_3300 Bachelor Thesis
-
-Das Templatet passt alle relevanten Einstellungen automatisch an, sobald der Document Type geändert wird.
-
-## Document Type spezifische Besonderheiten
-
-### T2_3100
-
-Die Studienarbeit ist eine reine Hochschularbeit. Aus diesem Grund wird der Ort der Firma und der Sperrvermerk entfernt. Desweiteren ist es möglich die Studienarbeit als Gruppe abzugeben. Hierfür gibt es die Variable multipleAuthors. Ist diese auf true gesetzt, passt sich die Eigenständigkeitserklärungs selbst von der Ich- zur Wir-Form an. Mehrere Autoren sind lediglich mit Komma getrent in die Variable documentAuthor einzutragen.
-
-# Komponenten einer Wissenschaftlichen Arbeit
+Studienarbeit des Studiengangs Informatik an der Dualen Hochschule Baden-Württemberg Heidenheim von Andreas Kiessling und Stefan Schneider
```

Comment:
Vielleicht nach dem jeweiligen Namen in Klammern noch auf das GitHub Profil verlinken?

Andreas Kiessling (@kiessla)